src/assets/sizes.js

```diff
@@ -1,3 +1,6 @@
-export default {
-  inputHeight: '40px'
-}
+export default {
+  inputHeight: '40px',
+  buttonMinWidth: '160px',
+  buttonMinHeight: '40px',
+  largeButtonMinWidth: '190px'
+}
```

Comment:
I saw in other frameworks a `controlHeight` property which is used for both input and normal buttons. Can't we refactor this like:
```suggestion

const BASE_SIZES = {
  controlHeight: '40px'
}

export default {
  ...BASE_SIZES,
  inputHeight: BASE_SIZES.controlHeight,
  buttonMinWidth: '160px',
  buttonMinHeight: BASE_SIZES.controlHeight,
  largeButtonMinWidth: '190px'
}
```

Reply:
Are you sure we have a button min width? Look [here](https://www.figma.com/file/7Wovqa84J9V3XSb6IzEUuK/IN-PROGRESS---Etvas-Styleguide?node-id=1058%3A115)

Reply:
Are you sure the button has a minHeight? Maybe it's buttonHeight?